src/query/formats/src/field_decoder/json_ast.rs

```diff
@@ -296,8 +297,8 @@ impl FieldJsonAstDecoder {
                 Ok(())
             }
             Value::Number(number) => match number.as_i64() {
-                Some(mut n) => {
-                    clamp_timestamp(&mut n);
+                Some(n) => {
+                    let n = int64_to_timestamp(n);
```

Comment:
Maybe other format need to do the same cast?

Reply:
done, thanks